### PR TITLE
cli: print zero balance of known token if `token` flag specified

### LINKED ISSUE #2313: Zero wallet balance string
Add explicit designation that a wallet is has `0` tokens while asking for a balance via `neo-go nep17 wallet balance` if `--token` flag used.

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #2313.

### CODE REVIEW

File: cli/wallet/nep17.go [hunk 1]
```diff
@@ -203,27 +206,86 @@ func getNEP17Balance(ctx *cli.Context) error {
...
-					amount = fixedn.ToString(b, tokenDecimals)
+			printAssetBalance(ctx, asset, tokenName, tokenSymbol, tokenDecimals, balances.Balances[i])
+		}
+		if name == "" || tokenFound {
+			continue
+		}
+		// Token was explicitly specified, but was not found among balances, thus either balance is 0
+		// or the token doesn't exists. Try to find token by its address/hash/name/symbol and print zero
+		// balance if found.
+		asset, err := flags.ParseAddress(name)
```
Comment (roman-khimov) [inline cli/wallet/nep17.go:224]: Why don't you `getMatchingToken` here? It can be specified by name and be present in the wallet metadata.

**Review by roman-khimov — COMMENTED** [review]
Maybe we can avoid going through nep17balances at all for a single known requested token? We just need to resolve name to hash (via wallet metadata or native NEP-17) and then ask the contract for a balance.

File: cli/wallet/nep17.go (hunk 1 above)
Comment (AnnaShaleva) [reply cli/wallet/nep17.go]: Fixed.

**Comment by AnnaShaleva** [comment]
> Maybe we can avoid going through nep17balances at all for a single known requested token? We just need to resolve name to hash (via wallet metadata or native NEP-17) and then ask the contract for a balance.

We also need to get the last updated block, and there's no way to get it except using the whole batch of `nep17balances` for this account, especially with P2PStateExchangeExtensions logic enabled.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**